zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -504,9 +504,9 @@ def convert_slack_workspace_messages(slack_data_dir: str, users: List[ZerverFiel
                                      added_users: AddedUsersT, added_recipient: AddedRecipientsT,
                                      added_channels: AddedChannelsT, realm: ZerverFieldsT,
                                      zerver_realmemoji: List[ZerverFieldsT], domain_name: str,
-                                     output_dir: str) -> Tuple[List[ZerverFieldsT],
-                                                               List[ZerverFieldsT],
-                                                               List[ZerverFieldsT]]:
+                                     output_dir: str, chunk_size: int=800) -> Tuple[List[ZerverFieldsT],
```

Comment:
Let's extract the default `chunk_size` to a constant that lives in `zerver.lib.export.py`, e.g. `MESSAGE_BATCH_CHUNK_SIZE = 1000` and import that from the things that use it.

Reply:
@timabott The `chunk_size` used in [export.py](https://github.com/zulip/zulip/blob/master/zerver/lib/export.py#L829) is 1000. But when I used the chunk_size as 1000 with the dataset of 15493 messages and 273 users(tried for gitter import), I had a successful gitter to zulip data conversion, but while importing, I received a Memory error. Working with `800` as the `chunk_size` didn't have any issue. I think it would be good if we decide upon a proper `chunk_size`. I have also mentioned this in #9569.

Reply:
@rheaparekh Do you mean Slack, rather than gitter? Also, you're focusing on the size, whereas I feel Tim was more just keen to have the constants factored-out; for now, these could even be different constants, until/if there's a consensus on a reasonable shared value.

Reply:
No, I meant gitter only. But both the scripts produce messages in the same manner, so it will be a problem for both. I can get the constants factored-out, but we need to decide upon an appropriate size.

Reply:
Added in 4906bfef7dc80232df439de5697ad73b90753062.